clouds/redshift/version

```diff
@@ -1 +1 @@
-1.1.1
+1.1.2
```

Comment:
Don't need to bump release only with docs. Please revert to 1.1.1 and we will keep the change only in the global changelog